tests/test_upload.py

```diff
@@ -90,9 +103,9 @@ def test_get_config_old_format(tmpdir):
         """))
 
     try:
-        upload.upload(dists="foo", repository="pypi", sign=None, identity=None,
-                      username=None, password=None, comment=None,
-                      sign_with=None, config_file=pypirc)
+        kwargs = default_kwargs.copy()
```

Comment:
If you prefer I can make the test fix a separate commit/PR, or rewrite it simpler (e.g.: add the new kwarg to upload call) ...